### DISCUSSION TITLE
I'm looking for an example Shell.TabBar handler

### DISCUSSION THREAD

**Discussion by HausBJB** [dotnet/maui#6979] [Q&A]
Hello, everyone,

I'm looking for an example to customize the TabBar in Shell.

e.g. like this
![image](https://user-images.githubusercontent.com/53478848/167560652-ce67abce-9c49-4e63-888b-63aed34a3249.png)

you could do it earlier with the renderer, e.g. like this

```
[assembly: ExportRenderer(typeof(CustomTabBar), typeof(CustomBottomBarRenderer))]
namespace XamFood.Droid.Renderers
{
    public class CustomBottomBarRenderer : TabbedPageRenderer
```

is there an example how to do this with MAUI Handlers using Shell?

Thanks for the help

**Comment by teslier** [comment]
Hi, have you found any @HausBJB ? I would like to set a different color for each icon, and potentially different icons when selected, unselected. Thank you!

**Comment by HausBJB** [comment]
No, unfortunately not. would have to keep the standard design

**Reply by teslier** [reply]
Ok, I let you know if I succeed implementing something out.

**Reply by teslier** [reply]
So, there is no handler for it in Microsoft.Maui.Handlers. You can do some basic modification like Shell.SetTabBarUnselectedColor(this, Colors.AliceBlue) in the AppShell.xaml.cs, but that's all I could find. 
I will skip it. I can use Shell.TabBarTitleColor, and Shell.TabBarUnselectedColor in <Tab> to achieve what I need.

**Comment by michaldivis** [comment]
I'd love to see a tabbar handler example, too.
